tests/c/interp_inside_call.c

```diff
@@ -0,0 +1,54 @@
+// Run-time:
+//   env-var: YKD_PRINT_IR=aot,jit-pre-opt
+//   env-var: YKD_SERIALISE_COMPILATION=1
+//   env-var: YKD_PRINT_JITSTATE=1
+//   stderr:
+//     jit-state: start-tracing
+//     ...
+//     jit-state: stop-tracing
+//     ...
+//     jit-state: enter-jit-code
+//     ...
+//     jit-state: stopgap
```

Comment:
can your next PR make this `enter-stopgap`?

Reply:
Sure.

Reply:
And add a `exit-stopgap` while we're there.

Reply:
Fixed in d3e1bd1ea